float32 lower_lim
float32 upper_lim
string filterFieldName
int8 negative
---

